CodeStarterCamp_Week4/bodycondition.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  bodycondition.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by MARY on 2023/03/14.
+//
+
+import Foundation
+
+class BodyCondition {
+    var upperBody: Int {
+        didSet {
+            upperBody > oldValue ?
+            print("상체근력이 \(upperBody-oldValue) 상승합니다.") :
+            print("상체근력이 \(oldValue-upperBody) 하락합니다.")
+        }
```

Comment:
좋습니다! 잘 활용해보셨군요! 👍👍👍
조금 더 깔끔하게 하려면~ 
하나의 상수로 선언하는것도 나쁘지 않아보이군요

아 그리고 스위프트에서는 연산자 앞뒤로 띄어쓰기를 넣어줍니다 ㅎㅎ

```suggestion
            let result = upperBody > oldValue ? "\(upperBody - oldValue) 상승" : "\(oldValue - upperBody) 하락"
            print("상체근력이 \(result)합니다.") 
        }
```

Reply:
상수명은.. 더 좋은 네이밍이 있겠지만.. 일단 예제니까.. 😅😅